mmeval/core/dist_backends/oneflow_dist.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+from typing import TYPE_CHECKING, Any, List, TypeVar
+
+from mmeval.utils import try_import
+from .base_backend import BaseDistBackend
+
+if TYPE_CHECKING:
+    import oneflow
+    import oneflow as flow
+    from oneflow.framework.check_point_v2 import _broadcast_py_object
+else:
+    flow = try_import('oneflow')
+    from oneflow.framework.check_point_v2 import _broadcast_py_object
```

Comment:
We should use `try_import` here, otherwise, the import could be failed if oneflow is not installed.

```suggestion
    check_point_v2 = try_import('oneflow.framework.check_point_v2')
```

And then we can use `check_point_v2._broadcast_py_object`.